src/components/Home/ReusableHomeSection.jsx

```diff
@@ -0,0 +1,55 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import { Box, Typography, Button } from '@mui/material';
+
+const ReusableHomeSection = ({ src, alt, title, description, button, href, label }) => {
```

Comment:
Just noticed this now, but this should be a round bracket/parenthesis `(` since it's doing an implicit return for the new component.

Reply:
This should fix the rendering problem that plagues this PR.